### ISSUE TITLE
Regarding training images

### ISSUE THREAD

**Issue by saravanan0206** [davidsandberg/facenet#336]
Hi,
 Please let me the format of the training images.  can we have the training images in .jpg format?

Thanks

**Comment by davidsandberg** [comment]
In the current repo (using TF 1.0) only png images are allowed. But if you are using a newer version of tensorflow you can change from using `decode_png` to `decode_image` in e.g. `train.softmax.py`:
```
                image = tf.image.decode_png(file_contents)
```

**Comment by davidsandberg** [comment]
Jpeg images are now supported (59844ee435f4e114a39a6b1aea59a5f609c32c39).
